VMEncryption/main/oscrypto/rhel_72_lvm/encryptstates/PatchBootSystemState.py

```diff
@@ -59,6 +59,9 @@ def enter(self):
 
         self.unmount_lvm_volumes()
         
+        # RHEL 7.8 does not seem to activate VG on close and open. 
+        # Hence, explicitly activating it.
+        self.command_executor.Execute('vgchange -a y rootvg')
```

Comment:
Is this the first time vgchange has been introduced?  If so we should make sure that it and the syntax used is valid across all supported RHEL + CentOS distros (E2E testing should cover).

Reply:
I believe vgchange is present for a while now. So far we didn't have to use it as luksClose and luksOpen seems to have been activating the volume group. This does not seem to be the case with RHEL 7.8. Hence, have to explicitly invoke vgchange to activate it. I tried the command on already activated VG and it didn't fail. Hence, thought it should be safe for older distros. But, I will definitely confirm it by running it through our AllDistros test.